### DISCUSSION TITLE
Intel 8085

### DISCUSSION THREAD

**Discussion by Mukhash2012** [SebLague/Digital-Logic-Sim#219] [Show and tell]
It's me again keeping discussions forum alive, and yes im pretty sure that there is also some opcode problems like the previous one.
But i will share it for you:
[8085.zip](https://github.com/user-attachments/files/18115347/8085.zip)
Opcodes:
[Opcodes](https://pastraiser.com/cpu/i8085/i8085_opcodes.html)
Data sheets:
[Data sheet](https://www.inf.pucrs.br/calazans/undergrad/orgcomp_EC/mat_microproc/intel-8085_datasheet.pdf)

I am now working on Intel 8088 and Intel 8086
Early Prototype of 8088 is ready:
[8088.zip](https://github.com/user-attachments/files/18115381/8088.zip)
![Снимок экрана 2024-12-12 223158](https://github.com/user-attachments/assets/ea30b434-2a75-4110-a406-06eb90aee08d)
![Снимок экрана 2024-12-12 223144](https://github.com/user-attachments/assets/aa4ca55a-e232-4cd0-998e-9b7cf2beaa2e)

**Comment by Mukhash2012** [comment]
8088 is here! Check for new discussions

**Reply by t0maszduda** [reply]
Hi, do You have new version 8085 for new Digital Logic Sim 2.1.6?

**Reply by Mukhash2012** [reply]
They are all already abandoned. You can still download 8008 tho for the new 2.1.6 which I made more accurate (remake)

**Reply by t0maszduda** [reply]
Where can I find it? on discussion there is one 8008 but is for old version 
`"ProjectName": "8008",
  "BuildVersion": "1.0.2",
  "CreationTime": "2023-02-02T14:59:41.031388-05:00",`

**Reply by Mukhash2012** [reply]
I will send you. Wait

**Reply by Mukhash2012** [reply]
Here is the i8008 - https://drive.google.com/file/d/1TZOyIZeJjnxg8suI-vlfrmhJpOOQ7Tdo/view?usp=drivesdk
Join also Discord servers for more stuff - https://discord.gg/RwhRJ8trH

**Reply by t0maszduda** [reply]
wow, thanks!!! Is it possible to make something similar to Altair computer with switches? We can program the computer using switches, exactly like in the Altair, for example a program like this: 
      
 
```
LDA 0080H    ; 00 111 010
             ; 10 000 000
             ; 00 000 000

MOV B,A      ; 01 000 111

LDA 0081H    ; 00 111 010
             ; 10 000 001
             ; 00 000 000

ADD B        ; 10 000 000

STA 0082H    ; 00 110 010
             ; 10 000 010
             ; 00 000 000

JMP 0000H    ; 11 000 011
             ; 00 000 000
             ; 00 000 000
```

**Reply by t0maszduda** [reply]
```
+------------------------------------------------------------------------------------------------------------------+
|                                                   ALTAIR 8800                                                    |
+------------------------------------------------------------------------------------------------------------------+

STATUS
[INTE] [PROT] [MEMR] [ INP] [  MI] [ OUT] [HLTA] [STACK] [  WO] [ INT]

DATA
 D7    D6    D5    D4    D3    D2    D1    D0
[●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]
 |     |     |     |     |     |     |     |
[^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]

ADDRESS
 A15   A14   A13   A12   A11   A10   A09   A08   A07   A06   A05   A04   A03   A02   A01   A00
 [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]   [●]
  |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]   [^]

CONTROL
[STOP/RUN]   [SINGLE STEP]   [   EXAMINE    ]   [   DEPOSIT    ]   [    RESET     ]
                             [ EXAMINE NEXT ]   [ DEPOSIT NEXT ]   [     CLR      ]

POWER
[OFF/ON]

+------------------------------------------------------------------------------------------------------------------+
```

**Reply by Mukhash2012** [reply]
Technically the recreation is accurate enough for you to programm it, i tested the commands, but remember because Sebastian Lague still didn't fix the instability bug the CPU might not start instantly and you need to reload the project.

**Reply by t0maszduda** [reply]
How can I find You on DLS Unofficial Server?

**Reply by Mukhash2012** [reply]
@null_0xf4

**Reply by t0maszduda** [reply]
Is there a chance to convert the 8080, 8085, and 8088 into the new DLS?

**Reply by Mukhash2012** [reply]
These projects are wildly inaccurate compared to real ones so I won't convert them

**Reply by t0maszduda** [reply]
Can we use your files (8085, 8008 etc.) to analyze on Discord and try to edit it and possibly expand it?

**Reply by Mukhash2012** [reply]
Ok
